frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -62,7 +63,10 @@ export const ObjectServiceDetailsCard: React.FC<DashboardItemProps> = ({
 
   const queryResult = prometheusResults.getIn([NOOBAA_SYSTEM_NAME_QUERY, 'result']);
 
-  const systemName = _.get(queryResult, 'data.result[0].metric.system_name');
+  const systemName = getMetric(queryResult, 'system_name');
+  const systemAddress = getMetric(queryResult, 'system_address');
+  const systemLink = `${systemAddress}fe/systems/${systemName}`;
+  const systemError = !systemName && !systemAddress;
```

Comment:
shouldn't it be like this ?
```suggestion
  const systemError = !systemName || !systemAddress;
```

Reply:
ack